{
    "Computer":  "SANSERV2019",
    "Events":  [
                   {
                       "TimeCreated":  "\/Date(1757639185742)\/",
                       "Source":  "VSS",
                       "LogName":  "Application",
                       "Level":  "Error",
                       "Message":  "Volume Shadow Copy Service error: Unexpected error DeviceIoControl(\\\\?\\GLOBALROOT\\Device\\HarddiskVolumeShadowCopy446 - 0000000000000350,0x0053c198,00000215ABDA3200,0,00000215ABDA6230,4096,[0]).  hr = 0x80070057, The parameter is incorrect.\r",
                       "Id":  12289,
                       "ErrorCode":  "0x0053c198",
                       "UpdateTitle":  null,
                       "EventRecordID":  208302
                   },
                   {
                       "TimeCreated":  "\/Date(1757638552516)\/",
                       "Source":  "Microsoft-Windows-TaskScheduler",
                       "LogName":  "Microsoft-Windows-TaskScheduler/Operational",
                       "Level":  "Error",
                       "Message":  "Task Scheduler failed to start instance \"{cc151c6b-42ac-4f10-a7a1-9d2dff7f62c3}\" of \"\\Microsoft\\Windows\\.NET Framework\\.NET Framework NGEN v4.0.30319\"  task for user \"NT AUTHORITY\\SYSTEM\" . Additional Data: Error Value: 2147946720.",
                       "Id":  103,
                       "ErrorCode":  {

                                     },
                       "UpdateTitle":  null,
                       "EventRecordID":  1027126
                   },
                   {
                       "TimeCreated":  "\/Date(1757638552516)\/",
                       "Source":  "Microsoft-Windows-TaskScheduler",
                       "LogName":  "Microsoft-Windows-TaskScheduler/Operational",
                       "Level":  "Error",
                       "Message":  "Task Scheduler failed to complete task \"\\Microsoft\\Windows\\.NET Framework\\.NET Framework NGEN v4.0.30319\" , instance \"{cc151c6b-42ac-4f10-a7a1-9d2dff7f62c3}\" , action \"\" . Additional Data: Error Value: 2147946720.",
                       "Id":  202,
                       "ErrorCode":  {

                                     },
                       "UpdateTitle":  null,
                       "EventRecordID":  1027125
                   },
                   {
                       "TimeCreated":  "\/Date(1757637859238)\/",
                       "Source":  "Service Control Manager",
                       "LogName":  "System",
                       "Level":  "Error",
                       "Message":  "The Update Orchestrator Service service hung on starting.",
                       "Id":  7022,
                       "ErrorCode":  {

                                     },
                       "UpdateTitle":  null,
                       "EventRecordID":  980812
                   },
                   {
                       "TimeCreated":  "\/Date(1757637823965)\/",
                       "Source":  "NETLOGON",
                       "LogName":  "System",
                       "Level":  "Error",
                       "Message":  "The session setup from the computer FO-2 failed to authenticate. The following error occurred: \r",
                       "Id":  5805,
                       "ErrorCode":  {

                                     },
                       "UpdateTitle":  null,
                       "EventRecordID":  980809
                   },
                   {
                       "TimeCreated":  "\/Date(1757637672948)\/",
                       "Source":  "NETLOGON",
                       "LogName":  "System",
                       "Level":  "Error",
                       "Message":  "The session setup from computer \u0027FO-2\u0027 failed because the security database does not contain a trust account \u0027FO-2$\u0027 referenced by the specified computer.  \r",
                       "Id":  5723,
                       "ErrorCode":  {

                                     },
                       "UpdateTitle":  null,
                       "EventRecordID":  980786
                   },
                   {
                       "TimeCreated":  "\/Date(1757637542203)\/",
                       "Source":  "Service Control Manager",
                       "LogName":  "System",
                       "Level":  "Error",
                       "Message":  "The SplashtopRemoteService service failed to start due to the following error: \r",
                       "Id":  7000,
                       "ErrorCode":  {

                                     },
                       "UpdateTitle":  null,
                       "EventRecordID":  980765
                   },
                   {
                       "TimeCreated":  "\/Date(1757637542203)\/",
                       "Source":  "Service Control Manager",
                       "LogName":  "System",
                       "Level":  "Error",
                       "Message":  "A timeout was reached (30000 milliseconds) while waiting for the SplashtopRemoteService service to connect.",
                       "Id":  7009,
                       "ErrorCode":  {

                                     },
                       "UpdateTitle":  null,
                       "EventRecordID":  980763
                   },
                   {
                       "TimeCreated":  "\/Date(1757637527129)\/",
                       "Source":  "Microsoft-Windows-DHCP-Server",
                       "LogName":  "System",
                       "Level":  "Error",
                       "Message":  "The DHCP service failed to see a directory server for authorization.",
                       "Id":  1059,
                       "ErrorCode":  {

                                     },
                       "UpdateTitle":  null,
                       "EventRecordID":  980748
                   },
                   {
                       "TimeCreated":  "\/Date(1757637512866)\/",
                       "Source":  "VSS",
                       "LogName":  "Application",
                       "Level":  "Error",
                       "Message":  "Volume Shadow Copy Service error: Unexpected error calling routine RegOpenKeyExW(-2147483646,SYSTEM\\CurrentControlSet\\Services\\VSS\\Diag,...).  hr = 0x80070005, Access is denied.\r",
                       "Id":  8193,
                       "ErrorCode":  "0x80070005",
                       "UpdateTitle":  null,
                       "EventRecordID":  208213
                   }
               ],
    "LastPoll":  "09/11 20:39"
}